server/seeders/20190115151310-demoUser.js

```diff
@@ -20,4 +20,4 @@ export default {
 ], {}),
 
   down: (queryInterface, Sequelize) => queryInterface.bulkDelete('Users', null, {})
-};
\ No newline at end of file
+};
```

Comment:
You could have amended the previous commit instead of making a new one just to add the new line.

Reply:
Okay